roles/horizon/defaults/main.yml

```diff
@@ -16,6 +16,7 @@ horizon:
       - { name: python-memcached, version: '1.48' }
       - { name: neutron-lbaas-dashboard }
       - { name: "git+https://github.com/blueboxgroup/horizon-customization#egg=horizon-customization"}
+      - { name: "git+https://github.com/blueboxgroup/django_openstack_auth@mitaka-bbg#egg=django_openstack_auth"} # for k2k federation
```

Comment:
Since this will break with constraints, can you try:

`- { name: django_openstack_auth, version: '2.1.1.dev1'  }`

That should pull from the newly built and mirrored build from our fork on our mirror.